src/App.js

```diff
@@ -5,10 +5,12 @@ import DetailsPage from './pages/DetailsPage'
 import BookmarkPage from './pages/BookmarkPage'
 import Navigation from './components/Navigation'
 import { Route, Switch, useHistory } from 'react-router-dom'
+import useLocalStorage from './utils/useLocalStorage'
```

Comment:
I would call the folder `hooks` instead of `utils` if `useLocalStorage` is a hook.

Reply:
Good tipp, thanx Lene!